src/coreclr/vm/clsload.cpp

```diff
@@ -2953,13 +2955,36 @@ TypeHandle ClassLoader::CreateTypeHandleForTypeKey(TypeKey* pKey, AllocMemTracke
     else if (pKey->GetKind() == ELEMENT_TYPE_FNPTR)
     {
         Module *pLoaderModule = ComputeLoaderModule(pKey);
```

Comment:
`ComputeLoaderModule` needs to be modified to take the custommods into account to handle the case where they are defined in unloaded assembly. 

Also, we need to call `EnsureInstantiation` or something like that to capture the dependencies.

Reply:
I think 'll hold off on this until we decide if we want to go with a restricted approach such as option (2) or (3) above since it wouldn't be needed assuming all calling conventions are in the runtime (e.g. System.Private.CoreLib) we don't have to worry about it being unloaded.